core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
     return findContext(contextName) != null;
   }
 
+  /* (non-Javadoc)
```

Comment:
I'll run some checks in the next day or two as I'm not 100% sure.


Sent by Outlook for Android





On Sun, Oct 25, 2015 at 7:21 PM -0700, "Mark Lewis" <notifications@github.com> wrote:
> @@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
>      return findContext(contextName) != null;
>    }
>
> +  /* (non-Javadoc)

Does the "non-Javadoc" comment still clear the javadoc warning?

---
Reply to this email directly or view it on GitHub:
https://github.com/psi-probe/psi-probe/pull/555/files#r42955364